Sources/Plasma/NucleusLib/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-add_definitions(-D_LIB)
+include_directories("${PLASMA_SOURCE_ROOT}/CoreLib")
+include_directories("${PLASMA_SOURCE_ROOT}/NucleusLib")
+include_directories("${PLASMA_SOURCE_ROOT}/NucleusLib/inc")
+include_directories("${PLASMA_SOURCE_ROOT}/PubUtilLib")
```

Comment:
TODO: Clean up anything from NucleusLib that depends on PubUtilLib classes

Reply:
Maybe it would be better to remove this at the top level and only include it in the subprojects that need it, with an eye towards removing it entirely?